### ISSUE TITLE
Possibility to add rounded corners to skeleton views

### ISSUE THREAD

**Issue by raphaklr** [Juanpe/SkeletonView#13]
Hi,

in the README, all screenshots show skeleton views with sharp corners. Is it possible to provide a corder radius so that the grey views have rounded corners?

**Comment by manuel21** [comment]
This featured will be awesome!

**Comment by Juanpe** [comment]
Hi @raphaklr and @manuel21, actually the round corners are allowed. If you view has corner the skeleton will draw with corner too. 

For example, with this code:
```swift
@IBOutlet weak var avatarImage: UIImageView! {
    didSet {
        avatarImage.layer.cornerRadius = avatarImage.frame.width/2
        avatarImage.layer.masksToBounds = true
    }
}
```
The result is:
<img width="317" alt="screen shot 2017-11-22 at 08 51 21" src="https://user-images.githubusercontent.com/1409041/33116025-76fc9f62-cf63-11e7-9fd3-dbbd2cdb4150.png">

**Comment by raphaklr** [comment]
@Juanpe thanks for the answer. But is it possible to have rounded corners in the skeleton view, while the actual views have sharp edges?

**Comment by Juanpe** [comment]
Mm...not, today it's not possible. I'm going to add this feature in the backlog, ok?
Thanks!

**Comment by B4V4-G** [comment]
Please check [PR 15](https://github.com/Juanpe/SkeletonView/pull/15)

**Comment by Juanpe** [comment]
Closed this issue, corner radius are available since version 1.2.1
